interface/main/onotes/office_comments_full.php

```diff
@@ -108,11 +109,14 @@
 $result_count = 0;
 foreach ($result as $iter) {
     $result_count++;
-    
+
+    $date=date( "Y-m-d" ,strtotime($iter{"date"}));
+    $date=oeFormatShortDate($date);
+
     if (getdate() == strtotime($iter{"date"})) {
-        $date_string = "Today, " . date( "D F dS" ,strtotime($iter{"date"}));
+        $date_string = xlt("Today, ") . $date;
```

Comment:
no need to change this. I already did it and brought your fix into the codebase :)
(above is just education)